assets/css/custom.css

```diff
@@ -0,0 +1,29 @@
+.title-header {padding: .75rem 1.25rem; background-color: #f5f5f5; border-bottom: 1px solid transparent;}
+.title-header h3 {font-size: 0.80rem; margin: 0;}
+.movies {margin-top: 2rem;}
```

Comment:
Why is `rem` better in this case? Or maybe it isn't, please tell why you chose this unit.

Reply:
Its not better.. :(  I got this template and i didn't improve it.